docs/manual_doc/Statement_Syntax/Data_Definition_Statements/Drop_Edge_Syntax.md

```diff
@@ -0,0 +1,6 @@
+```
+DROP EDGE edge_type_name
+```
+You must have the DROP privilege for the edge type.
```

Comment:
I think this sentence can be written better. Any ideas? @jude-zhu 
Privilege is needed to DROP the edge type.

Reply:
Refer to [Mysql](https://dev.mysql.com/doc/refman/8.0/en/create-table.html) , this sentence looks fine for me.